src/transforms/remap.rs

```diff
@@ -132,12 +136,18 @@ impl Remap {
     fn annotate_dropped(&self, event: &mut Event, reason: &str, error: ExpressionError) {
         match event {
             Event::Log(ref mut log) => {
+                let raw_message = error
+                    .labels()
+                    .pop()
+                    .expect("at least one label from error")
```

Comment:
What labels does an `error` have in this case?

Reply:
From my understanding, `labels` is populated with successive error messages from a function call and its potentially nested function calls: https://github.com/vectordotdev/vector/blob/master/lib/vrl/compiler/src/expression/function_call.rs#L228. With `pop()` we get the top most error message prior to it being interpolated into `"function call error..."`, but if there were nested calls that errored, even `raw_message` might contain `"function call error..."`.

Reply:
IMHO an odd name for what it contains, thanks!